concepticondata/concepticon.tsv

```diff
@@ -1673,7 +1673,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1672	WRITE	Speech and language	To form (letters, words or symbols) on a surface in order to communicate.	Action/Process	
 1673	ARM	The body	The upper limb, extending from the shoulder to the wrist and sometimes including the hand.	Person/Thing	
 1674	POOR	Possession	With little or no possessions or money.	Property	
-1675	PROMISE	Speech and language	A transaction between two persons whereby the first person undertakes in the future to render some service or gift to the second person or devotes something valuable now and here to his use.	Action/Process	
+1675	PROMISE	Speech and language	To assure someone that one will do something or that something will happen.	Action/Process	
```

Comment:
Not only `to assure` someone; for most societies, the elicitation probably picks something like "to commit".